tests/example_experiments_test.py

```diff
@@ -0,0 +1,95 @@
+# pylint: disable=invalid-name,no-self-use
+import pyhocon
+
+from deep_qa.common.params import Params, replace_none
+from deep_qa.models import concrete_models
+
+from .common.test_case import DeepQaTestCase
+
+
+class TestExampleExperiments(DeepQaTestCase):
+    def setUp(self):
+        super(TestExampleExperiments, self).setUp()
+        self.example_experiments_dir = "./example_experiments"
+        self.entailment_dir = self.example_experiments_dir + "/entailment/"
+        self.memory_networks_dir = self.example_experiments_dir + "/memory_networks/"
+        self.multiple_choice_qa_dir = self.example_experiments_dir + "/multiple_choice_qa/"
+        self.reading_comprehension_dir = self.example_experiments_dir + "/reading_comprehension/"
+        self.sentence_selection_dir = self.example_experiments_dir + "/sentence_selection/"
+        self.sequence_tagging_dir = self.example_experiments_dir + "/sequence_tagging/"
+
+    def test_entailment_examples_can_be_parsed(self):
```

Comment:
I would change all of these `can_be_parsed` names to `can_train`, as that's what we're actually doing now.